starter_code/routes/movies.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const router  = express.Router();
+const Movies = require('../models/Movie')
+
+
+router.get('/', (req, res, next) => {
```

Comment:
You can delete this whole file because is not connected with anything... all the routes are handled by this file `starter_code/routes/index.js`